### DISCUSSION TITLE
Asset ID {blah blah} was not found in the Cache

### DISCUSSION THREAD

**Discussion by steven-twerdochlib** [pixijs/pixijs#10781] [Q&A] [answered]
Hi,
I'm upgrading pixi from v7 to v8 but getting the error mentioned in title.
My code in v7 is the follow: 
```
// create the pixi renderer
const renderer = await PIXI.autoDetectRenderer({
  width: this.width,
  height: this.height,
  preserveDrawingBuffer: true,
  backgroundAlpha: 0
});
const renderTexture = PIXI.RenderTexture.create({ width: this.width, height: this.height });
const dataURL = await renderer.extract.base64(renderTexture);
renderTexture.destroy(true);
renderer.destroy();
this.sprite = PIXI.Sprite.from(dataURL);
```

**Comment by bigtimebuddy** [comment]
Please review the [migration guide for v8](https://pixijs.com/8.x/guides/migrations/v8) the section:

> Texture.from no longer will load a texture from a URL.

This also applies to `Sprite.from`. You need to use Assets to preload the image.

**Reply by steven-twerdochlib** [reply]
Hi, thanks for the quick reply, didn't realise it was for both sprite.from and texture.from, however my new line of code is returning a texture instead of a sprite? How do I fix that? Is there a specific loadParser I should add to my code:
`this.sprite = await PIXI.Assets.load({ src: dataURL });`

**Reply by bigtimebuddy** [reply, marked as answer]
Just add a new Sprite.
```ts
this.sprite = new PIXI.Sprite(await PIXI.Assets.load({ src: dataURL }));
```
